login-fire-button.html

```diff
@@ -198,13 +198,13 @@
      * The provider logo must be hidden if the element can sign out and a user
      * is currently signed in.
      *
-     * @param  {Boolean} onlySignIn is `false` if the element can sign out
+     * @param  {Boolean} noSignOut is `false` if the element can sign out
      * @param  {Boolean} signedIn   is `true` if a user is actually signed in
      * @return {Boolean}            `true` if the element can sign out and a
      * user is currently signed in.
      */
-    _computeHideIcon: function(onlySignOut, onlySignIn, signedIn) {
-      return onlySignOut || (!onlySignIn && signedIn);
+    _computeHideIcon: function(noSignIn, noSignOut, signedIn) {
+      return noSignIn || (!noSignOut && signedIn);
```

Comment:
My bad, since this is hiding it - it's the opposite of what I wrote. Since it should be shown then the hide value should be the opposite and therefore the return value should be `!((!noSignIn && !signedIn) || noSignOut)`.